### PR TITLE
(DPS-889) the new Platform Overview

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
The new Platform Overview is supposed to replace the existing Platform Overview. The content of the existing Platform Overview was divided in:

- **Platform Overview** (the most generic doc, something on top of all other documents, highlighting possible ways to extend LiveChat)

- Backend APIs (or Messaging APIs, the name not decided yet; the document which explains differences between Agent, Customer and Configuration API, as well as other concepts, such as chat routing, thread, chat, etc.)

- legacy blog post on differences between LC2 & LC3 (not created yet)

### CODE REVIEW

File: site/content@beta/platform-overview-new/about-livechat.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+weight: 30
+---
+
+# What LiveChat is
+
+**LiveChat** helps you handle conversations with the customers who visit your websites. Customers can start chats with your agents, but it can also be the other way round. Your agents can initiate chats, making your visitors feel welcome and valued. If you're up to some automation, you can engage them in a conversation with [**greetings**](https://www.livechatinc.com/help/why-should-i-set-up-chat-greetings/). Also, delegating some work to **bots** is an options when running a bussines on a bigger scale.
```
Comment (Klaym3n) [inline site/content@beta/platform-overview-new/about-livechat.md]: "bots" może odsyłać do artykułu: https://www.livechatinc.com/help/chatbots-explained/

File: site/content@beta/platform-overview-new/about-livechat.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+weight: 30
+---
+
+# What LiveChat is
+
+**LiveChat** helps you handle conversations with the customers who visit your websites. Customers can start chats with your agents, but it can also be the other way round. Your agents can initiate chats, making your visitors feel welcome and valued. If you're up to some automation, you can engage them in a conversation with [**greetings**](https://www.livechatinc.com/help/why-should-i-set-up-chat-greetings/). Also, delegating some work to **bots** is an options when running a bussines on a bigger scale. 
+
+You customers can start multiple conversations with agents, as well as continue the existing ones- it's up to them. Keeping track of chat history is easy, as agents can scroll up to get a preview of all the exchanged messages in a conversation.
```
Comment (Klaym3n) [inline site/content@beta/platform-overview-new/about-livechat.md]: "You customers" -> "Your customers"

Comment (Klaym3n) [inline site/content@beta/platform-overview-new/about-livechat.md]: "existing ones-" -> "existing ones -" (spacja po "ones")

File: site/content@beta/platform-overview-new/about-livechat.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+weight: 30
+---
+
+# What LiveChat is
+
+**LiveChat** helps you handle conversations with the customers who visit your websites. Customers can start chats with your agents, but it can also be the other way round. Your agents can initiate chats, making your visitors feel welcome and valued. If you're up to some automation, you can engage them in a conversation with [**greetings**](https://www.livechatinc.com/help/why-should-i-set-up-chat-greetings/). Also, delegating some work to **bots** is an options when running a bussines on a bigger scale. 
+
+You customers can start multiple conversations with agents, as well as continue the existing ones- it's up to them. Keeping track of chat history is easy, as agents can scroll up to get a preview of all the exchanged messages in a conversation. 
+
+All of these functionalities are guaranteed *out of the box*, but this is just a tiny bit of what LiveChat offers. If you're interested in other default features, check out our [**Knowledge Base**](https://www.livechatinc.com/features/).
```
Comment (Klaym3n) [inline site/content@beta/platform-overview-new/about-livechat.md]: Jest napisane "Knowledge Base", a prowadzi do Features. Also, Knowledge Base już nie istnieje, teraz jest to Help center: https://www.livechatinc.com/help/

Comment (ale-kacperczyk) [reply site/content@beta/platform-overview-new/about-livechat.md]: so true

File: site/content@beta/platform-overview-new/about-livechat.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+# What LiveChat is
+
+**LiveChat** helps you handle conversations with the customers who visit your websites. Customers can start chats with your agents, but it can also be the other way round. Your agents can initiate chats, making your visitors feel welcome and valued. If you're up to some automation, you can engage them in a conversation with [**greetings**](https://www.livechatinc.com/help/why-should-i-set-up-chat-greetings/). Also, delegating some work to **bots** is an options when running a bussines on a bigger scale. 
+
+You customers can start multiple conversations with agents, as well as continue the existing ones- it's up to them. Keeping track of chat history is easy, as agents can scroll up to get a preview of all the exchanged messages in a conversation. 
+
+All of these functionalities are guaranteed *out of the box*, but this is just a tiny bit of what LiveChat offers. If you're interested in other default features, check out our [**Knowledge Base**](https://www.livechatinc.com/features/). 
+
+You can also have a quick glance at the list below. It summarizes, from a more tech perspective, the features provided by our **LC3 protocol**:
```
Comment (ale-kacperczyk) [inline site/content@beta/platform-overview-new/about-livechat.md]: Jesli z zasady nie uzywamy slowa 'protocol' to moze zmienimy '**LC3** **protocol**' na **product**

File: site/content@beta/platform-overview-new/platform-overview.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+- Apps in the Details section (widgets)
+- Apps in the Customer Details section (Customer Detail widgets)
+- Chat Actions
+- Fullscreen apps displayed in the main menu
+- MessageBox
+
+Those are the possible integration interfaces that developers can work with. You can always check out our [Marketplace](https://www.livechatinc.com/marketplace/) to see what integrations are available for purchase.
+
+Each type of integration has its advantages, which in turn suggests use cases. The most common extensions are **apps in the Details section**, also known as widgets. They are quite versatile, which makes them usable in many cases. You could use them to display some statistics on your agents performance or to manage tags used by agents. Similar to **apps in Details** are the **apps in Customer Details**. There're perfect for displaying data needed at hand.
```
Comment (Klaym3n) [inline site/content@beta/platform-overview-new/platform-overview.md]: Brakuje screenshota appki w Details section

File: site/content@beta/platform-overview-new/platform-overview.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+Those are the possible integration interfaces that developers can work with. You can always check out our [Marketplace](https://www.livechatinc.com/marketplace/) to see what integrations are available for purchase.
+
+Each type of integration has its advantages, which in turn suggests use cases. The most common extensions are **apps in the Details section**, also known as widgets. They are quite versatile, which makes them usable in many cases. You could use them to display some statistics on your agents performance or to manage tags used by agents. Similar to **apps in Details** are the **apps in Customer Details**. There're perfect for displaying data needed at hand. 
+
+Speeding up the work can be easily achieved with **Chat Actions**, which are shortcuts to certain operations. Manual ticket creation takes a few steps, but with **Chat Actions** it's just a button click.
+
+![LiveChat Chat Actions](livechat-chat-actions.jpg)
+
+Web appplications can be displayed **fullscreen**, right in the Agent App. If you're planning on integrating your CRM with LiveChat, that might be the way to go.
```
Comment (Klaym3n) [inline site/content@beta/platform-overview-new/platform-overview.md]: Brak screenshota appki Fullscreenowej

File: site/content@beta/platform-overview-new/platform-overview.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+There's also another area where developers can integrate their apps- it's the Message Box. This interface can be used for extensions related strongly to messages, which are being sent.
+It could be used to send gifs (see [Integration with Giphy](https://developers.livechatinc.com/docs/building-apps-for-livechat/#messagebox)) or files from Dropbox.
```
Comment (Klaym3n) [inline site/content@beta/platform-overview-new/platform-overview.md]: To może prowadzić do pytania o integrację z Giphy, która nie istnieje na dzień dzisiejszy nigdzie publicznie.

File: site/content@beta/platform-overview-new/platform-overview.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+
+These two sets of APIs are powerful. The Agent Chat API lets you build your custom Agent App, as well as browser chat archives, or ban customers. With the use of the Customer Chat API, you can build a custom Chat Widget. Both APIs allow for interactions, such as joining a chat, posting messages, etc.
+
+## Configuration API
+
+The **Configuration API** is dedicated to handling licence and agent's properties, webhooks management, and bot agents configuration.
+
+# Interacting with data
+
+One of the most extensively used sections in the Agent App is **Reports**. We provide several types of reports, which help managers with the analysis. You can measure chat satisfaction and duration, keep track of agents' activity, or compare time periods to observe trends. Again, this is just a tiny bit of what's available in terms of reporting. Yet, we're aware that what we offer does not cover every single business case. This is where [**Reports API**](https://developers.livechatinc.com/docs/rest-api/#reports) comes into play. It serves as a sets of tools to create apps which interact on the server side. With this API, it's possible to design exactly what you need.
```
Comment (Klaym3n) [inline site/content@beta/platform-overview-new/platform-overview.md]: "One of the most extensively used sections in the Agent App is **Reports**" - haha, co? :D 
Anyway, podrzuciłbym w miejsce Reports link do Help Center: https://www.livechatinc.com/help/reporting-options-available-in-livechat/

File: site/content@beta/platform-overview-new/about-livechat.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+All of these functionalities are guaranteed *out of the box*, but this is just a tiny bit of what LiveChat offers. If you're interested in other default features, you can check them out[**here**](https://www.livechatinc.com/features/). 
+
+You can also have a quick glance at the list below. It summarizes, from a more tech perspective, the features provided by our product:
+
+|    | Feature |
+|----|--------|
+| ✅|text messaging APIs|
+| ✅|rich messaging APIs|
+| ✅|chat routing APIs|
+| ✅|asynchronous transport (Web API)|
```
Comment (bkubiak) [inline site/content@beta/platform-overview-new/about-livechat.md]: nie jestem pewny czy w ten sposób można powiedzieć o requestach XHR, wg mnie nie jest to asynchroniczny transport

File: site/content@beta/platform-overview-new/about-livechat.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+| ✅|real-time transport (websockets)|
+| ✅|continuous, threaded chat|
+| ✅|chatbots|
+| ✅|concurrent chats| 
+| ✅|custom chat events|
+| ✅|editable chat events|
```
Comment (bkubiak) [inline site/content@beta/platform-overview-new/about-livechat.md]: tego też nie jestem pewny czy się da - wydaje mi się, że nie

File: site/content@beta/platform-overview-new/about-livechat.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+| ✅|unlimited visitors tracking|
```
Comment (bkubiak) [inline site/content@beta/platform-overview-new/about-livechat.md]: to też nie jest prawda

File: site/content@beta/platform-overview-new/platform-overview.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+
+## Configuration API
+
+The **Configuration API** is dedicated to handling licence and agent's properties, webhooks management, and bot agents configuration.
+
+# Interacting with data
+
+One of the most extensively used sections in the Agent App is [**Reports**](https://www.livechatinc.com/help/reporting-options-available-in-livechat/). We provide several types of reports, which help managers with the analysis. You can measure chat satisfaction and duration, keep track of agents' activity, or compare time periods to observe trends. Again, this is just a tiny bit of what's available in terms of reporting. Yet, we're aware that what we offer does not cover every single business case. This is where [**Reports API**](https://developers.livechatinc.com/docs/rest-api/#reports) comes into play. It serves as a sets of tools to create apps which interact on the server side. With this API, it's possible to design exactly what you need.
```
Comment (skamieniarz) [inline site/content@beta/platform-overview-new/platform-overview.md]: I believe it should be: _in the Agent App **are** Reports_ considering the plural of sections and reports. Or _used *section* in the Agent App_ - singular section.

File: site/content@beta/platform-overview-new/about-livechat.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+
+You can also have a quick glance at the list below. It summarizes, from a more tech perspective, the features provided by our product:
+
+|    | Feature |
+|----|--------|
+| ✅|text messaging APIs|
+| ✅|rich messaging APIs|
+| ✅|chat routing APIs|
+| ✅|asynchronous transport (Web API)|
+| ✅|real-time transport (websockets)|
```
Comment (skamieniarz) [inline site/content@beta/platform-overview-new/about-livechat.md]: Are you referring here to websocket-like connection used in Agent and Customer APIs? If so, I wouldn't call it _real-time transport_. It connotes real-time transport protocol (RTP) which is rather used in streaming media or Voice over IP (e.g. video teleconferences). What we use is _real-time messaging_ (RTM).

File: site/content@beta/platform-overview-new/platform-overview.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+
+## Chat API
+
+Together, the **Agent Chat API** and the **Customer Chat API** create the **Chat API**. It is a full set of methods to use either as an agent or as a customer. We divide it though, to make it more use-case-specific. The distinction is intuitive; when you want to join a chat as an agent, use the Agent Chat API. To interact as a customer, use the Customer Chat API. 
+
+These two sets of APIs are powerful. The Agent Chat API lets you build your custom Agent App, as well as browser chat archives, or ban customers. With the use of the Customer Chat API, you can build a custom Chat Widget. Both APIs allow for interactions, such as joining a chat, posting messages, etc.
+
+## Configuration API
+
+The **Configuration API** is dedicated to handling licence and agent's properties, webhooks management, and bot agents configuration.
```
Comment (skamieniarz) [inline site/content@beta/platform-overview-new/platform-overview.md]: Configuration API is not really dedicated to handling licence and agent's properties. It's used for storing configuration of license which means creating properties, but possible values for `<property_name>.locations.<location>` while creating a property are: chat, thread or event.

Also, as you've mentioned, Configuration API is used for webhooks and bot agents management (registering  and unregistering webhooks, creating and removing bot agents and so on).

File: site/content@beta/platform-overview-new/platform-overview.md [hunk 15]
```diff
@@ -0,0 +1,80 @@
...
+- [Agent Chat API](https://developers.livechatinc.com/beta-docs/agent-chat-api/)
+- [Customer Chat API](https://developers.livechatinc.com/beta-docs/customer-chat-api/)
+
+There's also the [Configuration API](https://developers.livechatinc.com/beta-docs/configuration-api/).
+
+## Chat API
+
+Together, the **Agent Chat API** and the **Customer Chat API** create the **Chat API**. It is a full set of methods to use either as an agent or as a customer. We divide it though, to make it more use-case-specific. The distinction is intuitive; when you want to join a chat as an agent, use the Agent Chat API. To interact as a customer, use the Customer Chat API. 
+
+These two sets of APIs are powerful. The Agent Chat API lets you build your custom Agent App, as well as browser chat archives, or ban customers. With the use of the Customer Chat API, you can build a custom Chat Widget. Both APIs allow for interactions, such as joining a chat, posting messages, etc.
```
Comment (skamieniarz) [inline site/content@beta/platform-overview-new/platform-overview.md]: That statement sounds a little weird. What do you mean by _powerful_? Considering possibility of creating your own implementations (Agent App, Chat Widget), perhaps sth like _These two sets of APIs are very versatile_ would be more accurate?

**Pull request merged**